msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -213,7 +213,35 @@ A full width Adaptive Card appears as follows:
 If you have not set the `width` property to *Full*, then the default view of the Adaptive Card is as follows:
 ![Small width Adaptive Card view](../../assets/images/cards/small-width-adaptive-card.png)
 
+### Stage View for images in Adaptive cards
+You can also use the `msteams` property to selectively add the ability to enlarge images, in an Adaptive card, in stage view. Users would see an expand icon when they hover over the images, for which the `allowExpand` attribute is set to `true`. For information on how to use the property, see the following example:
```

Comment:
```suggestion
In an Adaptive card, in stage view, you can use the `msteams` property to add the ability to enlarge images selectively. When users hover over the images, they would see an expand icon, for which the `allowExpand` attribute is set to `true`. For information on how to use the property, see the following example:
```

Reply:
this property is added  which enables users to see the image in stage view. Updated to reflect the same. please review and let me know if this doesnt look right.